projects/core/src/cart/store/multi-cart-store.module.ts

```diff
@@ -1,30 +1,15 @@
 import { CommonModule } from '@angular/common';
 import { NgModule } from '@angular/core';
+import { EffectsModule } from '@ngrx/effects';
 import { StoreModule } from '@ngrx/store';
 import { StateModule } from '../../state/state.module';
+import { MultiCartEffects } from './effects/multi-cart.effect';
 import { MULTI_CART_FEATURE } from './multi-cart-state';
 import {
+  multiCartMetaReducers,
   multiCartReducerProvider,
   multiCartReducerToken,
-  multiCartMetaReducers,
 } from './reducers/index';
-import { ConfigModule } from '../../config/config.module';
-import { StorageSyncType, StateConfig } from '../../state/config/state-config';
-import { MultiCartEffects } from './effects/multi-cart.effect';
-import { EffectsModule } from '@ngrx/effects';
-
-export function multiCartStoreConfigFactory(): StateConfig {
-  const config: StateConfig = {
-    state: {
-      storageSync: {
-        keys: {
-          [`${MULTI_CART_FEATURE}.active`]: StorageSyncType.LOCAL_STORAGE,
```

Comment:
Can you help me understand why we're not touching those guys https://github.com/SAP/cloud-commerce-spartacus-storefront/blob/b031b29d165439050e8852c0d84be3414690354a/projects/core/src/cart/store/cart-store.module.ts#L28-L30

Reply:
It will be removed as part of cart deprecations removal.